Makefile

```
@@ -59,23 +75,9 @@ fix-imports:
 	isort -rc ./app ./tests
 
 .PHONY: freeze-requirements
-freeze-requirements:
-	rm -rf venv-freeze
-	virtualenv -p python3 venv-freeze
-	$$(pwd)/venv-freeze/bin/pip install -r requirements-app.txt
-	echo '# pyup: ignore file' > requirements.txt
-	echo '# This file is autogenerated. Do not edit it manually.' >> requirements.txt
-	cat requirements-app.txt >> requirements.txt
-	echo '' >> requirements.txt
-	$$(pwd)/venv-freeze/bin/pip freeze -r <(sed '/^--/d' requirements-app.txt) | sed -n '/The following requirements were added by pip freeze/,$$p' >> requirements.txt
-	rm -rf venv-freeze
-
-.PHONY: test-requirements
-test-requirements:
-	@diff requirements-app.txt requirements.txt | grep '<' \
-	    && { echo "requirements.txt doesn't match requirements-app.txt."; \
-	         echo "Run 'make freeze-requirements' to update."; exit 1; } \
-|| { echo "requirements.txt is up to date"; exit 0; }
+freeze-requirements: requirements-for-test requirements.in requirements_for_test.in
```

actually, we can get rid of pip-tools from requirements-for-test, and then make this command much quicker and more self-contained by just installing pip-tools within the command and running something like:

```suggestion
freeze-requirements:  ## create static requirements.txt 
	${VIRTUALENV_ROOT}/bin/pip install pip-tools
	${VIRTUALENV_ROOT}/bin/pip-compile requirements.in
	${VIRTUALENV_ROOT}/bin/pip-compile requirements_for_test.in
```

---

Changed in c34a605